packages/widgetbook/lib/src/addons/addon_injector_widget.dart

```diff
@@ -12,23 +12,23 @@ class AddonInjectorWidget extends StatelessWidget {
 
   final List<WidgetbookAddOn> addons;
   final Widget child;
-  final Map<String, dynamic> routerData;
+  final Map<String, String> routerData;
 
   @override
   Widget build(BuildContext context) {
     return Nested(
       key: ValueKey(routerData),
-      children: addons
-          .map(
-            (e) => SingleChildBuilder(
-              builder: (context, child) => e.wrapperBuilder(
-                context,
-                routerData,
-                child!,
-              ),
+      children: [
```

Comment:
Does this allow for an empty `addons` list?
When I tried creating a Widgetbook app with an empty `addons` list in the latest beta release of widgetbook this widget threw because the `Nested` widget does not allow for an empty `children` list. Should the injector widget return an empty Container instead of `Nested` for an empty `addons` list to resolve this?

Reply:
Good point.

Reply:
Created #522 will fix in another PR